src/applications/vaos/components/CancelAppointmentModal.jsx

```diff
@@ -21,6 +25,52 @@ export default class CancelAppointmentModal extends React.Component {
       return null;
     }
 
+    const isVideo = isVideoVisit(appointmentToCancel);
+    const isCC =
+      getAppointmentType(appointmentToCancel) ===
+      APPOINTMENT_TYPES.ccAppointment;
+
+    if (isVideo || isCC) {
+      const facilityName = isVideo
```

Comment:
We aren't going to know the facility or phone for video appointments, I don't think. We get a facilityId in the appointment record that's really the system id, so we don't know exactly where the user scheduled their video call.